index.html

```diff
@@ -216,5 +216,82 @@ <h2 class="aboutTitle">Skills</h2>
         </li>
       </ul>
     </section>
+    <section class="contact">
+      <div class="contactHeader">
+        <h2 class="contactTitle">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+      </div>
+
+      <form
+        class="contactForm"
+        action="https://formspree.io/f/mknarzzq"
+        method="post"
+      >
+        <input
+          id="fullName"
+          name="fullName"
+          type="text"
+          placeholder="Full Name"
+          required
+        />
+        <img
+          class="fieldSeparator"
+          src="/images/separatorForm.svg"
+          alt="separator"
+        />
+        <input
+          id="email"
+          name="email"
+          type="email"
+          placeholder="Email address"
+        />
```

Comment:
- [ ] According to the requirements, all the `input` fields should be `required` 😀, I suggest that you implement these changes. 🙂.